packages/conf-ncurses/conf-ncurses.1/opam

```diff
@@ -7,6 +7,9 @@ license: "MIT"
 build: [
   ["pkg-config" "ncurses"] { os != "darwin"}
 ]
+depends: [
+  "conf-pkg-config" { os != "darwin"}
```

Comment:
Sure, probably better to wait for the next update of ocamlfind to fix this. Unless the confusion in https://github.com/diml/utop/issues/159 pops up again.